rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1377,10 +1377,10 @@ public BooleanWrapper setNodeSourcePingFrequency(int frequency, String sourceNam
     }
 
     /**
-     * {@inheritDoc}
+     * Gives list of existing Node Sources
+     * @return list of existing Node Sources
      */
-    @Deprecated
-    public List<RMNodeSourceEvent> getNodeSourcesList() {
+    public List<RMNodeSourceEvent> getExistingNodeSourcesList() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)